src/utils/I18nUtils.ts

```diff
@@ -0,0 +1,9 @@
+const pluralRules = new Intl.PluralRules("en-US");
+
+export function pluralize(count: number, singular: string, plural?: string, skipCountInReturnedValue = false): string {
+  const pluralRule = pluralRules.select(count);
+  if (pluralRule === "one") {
+    return `${!skipCountInReturnedValue ? `${count} ` : ""}${singular}`;
```

Comment:
The outer layer of string interpolation is unneeded and just makes things more confusing IMO